### ISSUE TITLE
Android app worldwide availability

### ISSUE THREAD

**Issue by Krusty93** [nhsx/covid-19-app-android-ag-public#4]
Hello,
I'm wondering why the Android app version has been published only on UK store. Foreign people may have lot of troubles trying to downloading the app because usually everyone tends to keep its own country into the Play Store.
Other contact-tracing apps like the Italian Immuni, are available everywhere.
I am aware that iOS is able to understand in which region the user is and automatically manage different apps, instead Android needs user input, but this is a very big issue for both foreign people in UK and who's coming from abroad to spend a short time in UK.

I remember the Play Store allows the user to change the country only one time per year.

Thanks.

**Comment by ataraxia89** [comment]
The desire for a worldwide app has been acknowledged and addressed in the NHS support pages here >> https://faq.covid19.nhs.uk/article/KA-01127/en-us?parentid=CAT-01031&rootid=CAT-01023

If this does come to fruition it will be handled via a separate app as specified, therefore @Krusty93 this issue should be closed.

**Comment by Krusty93** [comment]
Hello, thank you for interesting.
However, from the linked website: "to create an app that can be used even while travelling in other countries". This is not what I've asked for.

I've asked to make the app available for foreign people who lives in the UK but they have another country selected on their smartphones. Google allows the country switch one time per year, so downloading the app can be very very tricky for these people. To give you an example, I live in the UK and in my Italian friend circle only one person out of 10 was able to download the app due to this limitation on their Android phones.
 
You can achieve the worldwide distribution solving this issue simply checking the "all countries" checkbox in the PlayStore console.

Distributing another app for foreign countries is another thing, much more complicated. Moreover, I don't think it is worth the effort since each country has its own app.

**Comment by benpryke** [comment]
@Krusty93 In lieu of Play Store access, I believe you can still install the app directly via the apk file. You can find downloads for this file from play store apk mirror sites.

**Comment by Krusty93** [comment]
Of course, you can but this is not a solution yet. You are forcing users to download an app from untrusted sources and changing default phone security settings. Plus, this is unknown for most of the users.
Finally they wouldn't get updates neither.

**Comment by AdamGleave** [comment]
+1 to this issue. I usually live in the US, but am currently resident in the UK. If I were to change my Play store region to the UK I'd lose access to various US-only apps that I need (e.g. banking). I was able to resolve this by creating a second Google account with UK region and adding both to my Android phone -- but this is not something most users would know is a possibility.

I see little harm with making this app globally available. If there are concerns with abuse or accidental installation, you might be able to verify UK location via GPS or cell tower information.
